examples/hostpath-provisioner/hostpath-provisioner.go

```diff
@@ -24,7 +24,7 @@ import (
 	"path"
 	"syscall"
 
-	"sigs.k8s.io/sig-storage-lib-external-provisioner/v8/controller"
+	"sigs.k8s.io/sig-storage-lib-external-provisioner/v9/controller"
```

Comment:
There are more references to `v8` in the `go.mod` file in this directory.

Reply:
Fixed